.drone.yml

```diff
@@ -210,10 +243,47 @@ trigger:
   branch:
     - "staging"
     - "staging-next"
+    - "seth/fake-staging"
   event:
     - pull_request
+
+---
+
+# On push, cache a clone of staging to drone's S3 bucket.
+# The staging cache reduces git bandwidth in PR pipelines.
+kind: pipeline
+type: docker
+name: cache-staging-clone
+
+clone:
+  disable: true
+
+steps:
+
+  - name: clone
+    image: bitnami/git
+    commands:
+      - git lfs install
+      - git clone --branch $DRONE_BRANCH $DRONE_GIT_HTTP_URL . 
+
+  - name: cache-staging-clone
+    image: plugins/s3-cache
+    settings:
+      filename: cache-staging-clone.tar
+      endpoint: s3://code-dot-org-drone
+      rebuild: true
+      debug: true
+      pull: true
+      mount:
+        - .
+
+trigger:
+  branch:
+    - "seth/fake-staging"
+  event:
+    - push
```

Comment:
nit: move `trigger` to be right after `kind`, `type`, and `name`. Then the comment become less necessary.

Reply:
We're keeping the same ordering of the keys as the other pipelines, discussed with @sureshc